### DISCUSSION TITLE
欢迎来到 ISZY工具集合 讨论区!

### DISCUSSION THREAD

**Discussion by ZvonimirSun** [ZvonimirSun/iszy-tools#123] [公告]
## 👋 欢迎！
  我希望能够在这里与使用者进行交流，了解用户的需求。大家可以在这里：
  * 咨询使用上的问题。
  * 分享你对工具集合的想法和建议。
  * 提出你的工具需求

**Comment by starfish0422** [comment]
多次提前还贷计算器 的 "贷款期限"栏 选中 "自定义贷款期限"后 只要一修改贷款月数,就不会进行计算了,所有结果为空!
不知道是否为bug?

**Reply by ZvonimirSun** [reply]
听起来感觉是的，这个我有空看看，重构优化下

**Reply by ZvonimirSun** [reply]
改掉啦

**Reply by starfish0422** [reply]
是的,可以了,
给力!!!

**Comment by Alysawei** [comment]
多次提前还贷计算器 的 还款方式，可以增加一个 自由还款嘛？比如公积金是可以选择自由还款的

**Reply by ZvonimirSun** [reply]
我没有公积金还款过，不是很了解，有空我看看
